source/Account.cpp

```diff
@@ -341,6 +349,14 @@ void Account::AddFine(int64_t amount)
 
 
 
+// Unpaid profit shares have a medium term and a low interest rate.
+void Account::AddProfitShares(int64_t bonus)
+{
+	mortgages.emplace_back(bonus, 1000, 90);
+}
```

Comment:
```suggestion
void Account::AddProfitShares(int64_t amount)
{
	mortgages.emplace_back(amount, 1000, 90);
}
```

Just for consistency with the other variable names in this file.

Reply:
I did wonder at that when I saw it in MZ's code.  I didn't touch it since that's how he did it, but if you say it's safe yeah your way makes more sense to me.